Do not build test programs

Biulding test programs does not work when we want to do a static link,
because duma.a redefines memcpy and strcpy, so the link fails.

Signed-off-by: "Yann E. MORIN" <yann.morin.1998@free.fr>

diff -durN duma-2.5.15.orig/GNUmakefile duma-2.5.15/GNUmakefile
--- duma-2.5.15.orig/GNUmakefile	2014-11-16 14:47:05.874448560 +0100
+++ duma-2.5.15/GNUmakefile	2014-11-16 14:54:50.792048921 +0100
@@ -294,7 +294,7 @@
 SO_OBJECTS=dumapp_so.o duma_so.o sem_inc_so.o print_so.o
 
 # Make all the top-level targets the makefile knows about.
-all:	libduma.a tstheap$(EXEPOSTFIX) dumatest$(EXEPOSTFIX) thread-test$(EXEPOSTFIX) testmt$(EXEPOSTFIX) dumatestpp$(EXEPOSTFIX) testoperators$(EXEPOSTFIX) $(DUMA_DYN_DEPS)
+all:	libduma.a $(DUMA_DYN_DEPS)
 
 # Perform self tests on the program this makefile builds.
 check test:
